build-test.xml

```diff
@@ -5090,43 +5090,43 @@ go]]></replacevalue>
-					mvn archetype:generate --batch-mode -DarchetypeArtifactId=com.liferay.project.templates.npm.portlet -DarchetypeGroupId=com.liferay -DarchetypeVersion=1.0.8 -DartifactId=com.npm.simple -Dauthor=liferay -DbuildType=maven -DclassName=Foo -DgroupId=com.test -DinteractiveMode=false -DliferayVersion=7.1 -Dpackage=com.test -DprojectType=standalone -Dversion=1.0.0
+					../../../../gradlew generateSampleProject
 				</execute>
 
-				<execute dir="${project.dir}/modules/sdk/project-templates/project-templates-npm-portlet/com.npm.simple">
-					mvn package
+				<execute dir="${project.dir}/modules/sdk/project-templates/project-templates-npm-portlet/build/com.npm.simple">
+					../../../../../../gradlew deploy
 				</execute>
 
 				<copy
-					file="${project.dir}/modules/sdk/project-templates/project-templates-npm-portlet/com.npm.simple/target/com.npm.simple-1.0.0.jar"
+					file="${project.dir}/modules/sdk/project-templates/project-templates-npm-portlet/build/bundles/osgi/modules/com.npm.simple.jar"
 					todir="${liferay.home}/deploy"
 				/>
 			</then>
 			<else>
 				<if>
-					<available file="${project.dir}/modules/sdk/project-templates/project-templates-npm-${blade.templates}-portlet/com.npm.${blade.templates}" type="dir" />
+					<available file="${project.dir}/modules/sdk/project-templates/project-templates-npm-${blade.templates}-portlet/build" type="dir" />
 					<then>
-						<delete dir="${project.dir}/modules/sdk/project-templates/project-templates-npm-${blade.templates}-portlet/com.npm.${blade.templates}" />
+						<delete dir="${project.dir}/modules/sdk/project-templates/project-templates-npm-${blade.templates}-portlet/build" />
 					</then>
 				</if>
 
 				<execute dir="${project.dir}/modules/sdk/project-templates/project-templates-npm-${blade.templates}-portlet">
-					mvn archetype:generate --batch-mode -DarchetypeArtifactId=com.liferay.project.templates.npm.${blade.templates}.portlet -DarchetypeGroupId=com.liferay -DarchetypeVersion=1.0.8 -DartifactId=com.npm.${blade.templates} -Dauthor=liferay -DbuildType=maven -DclassName=Foo -DgroupId=com.test -DinteractiveMode=false -DliferayVersion=7.1 -Dpackage=com.test -DprojectType=standalone -Dversion=1.0.0
+					../../../../gradlew generateSampleProject
 				</execute>
 
-				<execute dir="${project.dir}/modules/sdk/project-templates/project-templates-npm-${blade.templates}-portlet/com.npm.${blade.templates}">
-					mvn package
+				<execute dir="${project.dir}/modules/sdk/project-templates/project-templates-npm-${blade.templates}-portlet/build/com.npm.${blade.templates}">
+					../../../../../../gradlew deploy
 				</execute>
 
 				<copy
-					file="${project.dir}/modules/sdk/project-templates/project-templates-npm-${blade.templates}-portlet/com.npm.${blade.templates}/target/com.npm.${blade.templates}-1.0.0.jar"
+					file="${project.dir}/modules/sdk/project-templates/project-templates-npm-${blade.templates}-portlet/build/bundles/osgi/modules/com.npm.${blade.templates}.jar"
```

Comment:
This would change the location of the jar to `project-templates-npm-${blade.templates}-portlet/build/libs/com.npm.${blade.templates}.jar`